YgoProPatcher/Client.cs

```diff
@@ -510,7 +509,7 @@ private void DownloadUsingCDB(List<string> listOfDownloadedCDBS, string destinat
 
         private async Task GitHubDownload(string destinationFolder)
         {
-            Status.Invoke(new Action(() => { Status.Text = "Updating CDBS from Live2017Links"; }));
+            Status.Invoke(new Action(() => { Status.Text = "Updating card databases from Live2017Links."; }));
```

Comment:
CDBs instead.

Reply:
It would help new users know what a CDB is.